index.html

```diff
@@ -30,6 +30,247 @@ <h3 class="supporting-text">I'am a Software Developer</h3>
             <img src="./Microverse-Student-Project-1(4)/Tertiary/Disabled.png" alt="Medium icon">   
         </div>    
     </section>
+                <!--Works Section-->
+
+    <div class="grid-container">
+        <header class="title">
+            <h1 class="title-h1">My Recent Works</h1>
+            <hr class="title-hr">
+            <div class="frame">
+                <div class="featured">
+                   <div class="right-block">
+                        <img class="feature-img" src="./images/ImgPlaceholder.png" alt="">
+                        <h2 class="title-post">Multi-Post Stories</h2>
+                        <p class="feature-supporting-text">
+                            A daily selection of privately personalized
+                            reads; no accounts or sign-ups required.
+                            has been the industry's standard dummy
+                            text ever since the 1500s, when an
+                            unknown printer took a standard dummy text.
+                        </p>
+                        <div class="tags">
+                            <ul class="tag1">
+                                <li>css</li>
+                            </ul>
+                            <ul class="tag2">
+                                <li>html</li>
+                            </ul>
+                            <ul class="tag3">
+                                <li>bootstrap</li>
+                            </ul>
+                            <ul class="tag4">
+                                <li>Ruby</li>
+                            </ul>
+                        </div>
+                        <button type="submit" class="button">See Project</button>
+                    </div>
```

Comment:
- [ ] `required`

- [ ] please add a hover to your button see this [article](https://www.w3schools.com/csSref/sel_hover.asp) to make one